tests/time_tracker_api/activities/activities_namespace_test.py

```diff
@@ -57,6 +58,21 @@ def test_list_all_activities(client: FlaskClient, mocker: MockFixture):
     repository_find_all_mock.assert_called_once()
 
 
+def test_list_all_activities_with_conditions(client: FlaskClient, mocker: MockFixture):
+    from time_tracker_api.activities.activities_namespace import activity_dao
+    repository_find_all_mock = mocker.patch.object(activity_dao.repository,
+                                                   'find_all',
+                                                   return_value=[])
+
+    response = client.get("/activities?a=b", follow_redirects=True)
```

Comment:
This is not taking into account that the attributes of the filter are not the unique URL params that are passed. Other params can be, for instance, order attributes and pagination variables (offset, pageSize). That is why in #91 we proposed to enclose those filter attributes in an array, like `filter`.